contracts/StandardToken.sol

```diff
@@ -60,7 +60,7 @@ contract StandardToken is ERC20, SafeMath {
     //  allowance to zero by calling `approve(_spender, 0)` if it is not
     //  already 0 to mitigate the race condition described here:
     //  https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729
-    if ((_value != 0) && (allowed[msg.sender][_spender] != 0)) throw;
+    require((_value == 0) || (allowed[msg.sender][_spender] == 0));
```

Comment:
This will be same as above in ReleasableToken.sol as using two separate require()s here will result in "and" instead of "or" logical operator which will be incorrect.